<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>setAFlightPath</class>
 <widget class="QDialog" name="setAFlightPath">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>698</width>
    <height>296</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>输入航路参数</string>
  </property>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>273</x>
     <y>118</y>
     <width>149</width>
     <height>116</height>
    </rect>
   </property>
   <property name="title">
    <string>初始速度(m/s)</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>vx:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QDoubleSpinBox" name="spinvx">
      <property name="decimals">
       <number>2</number>
      </property>
      <property name="minimum">
       <double>-9999.000000000000000</double>
      </property>
      <property name="maximum">
       <double>9999.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
      <property name="value">
       <double>20.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>vy:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDoubleSpinBox" name="spinvy">
      <property name="decimals">
       <number>2</number>
      </property>
      <property name="minimum">
       <double>-9999.000000000000000</double>
      </property>
      <property name="maximum">
       <double>9999.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
      <property name="value">
       <double>0.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>vz:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QDoubleSpinBox" name="spinvz">
      <property name="decimals">
       <number>2</number>
      </property>
      <property name="minimum">
       <double>-9999.000000000000000</double>
      </property>
      <property name="maximum">
       <double>9999.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
      <property name="value">
       <double>0.000000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_6">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>49</y>
     <width>160</width>
     <height>58</height>
    </rect>
   </property>
   <property name="title">
    <string>目标类型</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_6">
    <item row="0" column="1">
     <widget class="QRadioButton" name="radioHelicopter">
      <property name="text">
       <string>直升机</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QRadioButton" name="radioPlane">
      <property name="text">
       <string>飞机</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_8">
   <property name="geometry">
    <rect>
     <x>332</x>
     <y>49</y>
     <width>141</width>
     <height>60</height>
    </rect>
   </property>
   <property name="title">
    <string>运动类型</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_8">
    <item row="0" column="0">
     <widget class="QComboBox" name="cmbType"/>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>49</y>
     <width>145</width>
     <height>58</height>
    </rect>
   </property>
   <property name="title">
    <string>是否为敌方</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QRadioButton" name="radioenemy">
      <property name="text">
       <string>敌方</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QRadioButton" name="radiooos">
      <property name="text">
       <string>己方</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>118</y>
     <width>117</width>
     <height>116</height>
    </rect>
   </property>
   <property name="title">
    <string>初始位置(m)</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="1" column="1">
     <widget class="QDoubleSpinBox" name="spinlocy">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="accelerated">
       <bool>true</bool>
      </property>
      <property name="minimum">
       <double>-99999.000000000000000</double>
      </property>
      <property name="maximum">
       <double>99999.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QDoubleSpinBox" name="spinlocz">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimum">
       <double>-99999.000000000000000</double>
      </property>
      <property name="maximum">
       <double>99999.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QDoubleSpinBox" name="spinlocx">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
      <property name="wrapping">
       <bool>true</bool>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::UpDownArrows</enum>
      </property>
      <property name="correctionMode">
       <enum>QAbstractSpinBox::CorrectToPreviousValue</enum>
      </property>
      <property name="minimum">
       <double>-99999.000000000000000</double>
      </property>
      <property name="maximum">
       <double>99999.000000000000000</double>
      </property>
      <property name="value">
       <double>10.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>x:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_5">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>y:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_4">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>z:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>429</x>
     <y>118</y>
     <width>117</width>
     <height>116</height>
    </rect>
   </property>
   <property name="title">
    <string>加速度(m/s^2)</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>ax</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QDoubleSpinBox" name="spinax"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>ay</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDoubleSpinBox" name="spinay"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>az</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QDoubleSpinBox" name="spinaz"/>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>137</x>
     <y>118</y>
     <width>126</width>
     <height>116</height>
    </rect>
   </property>
   <property name="title">
    <string>各方向运动类型</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>x</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="cmbtypex"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>y</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="cmbtypey"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>z</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="cmbtypez"/>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_7">
   <property name="geometry">
    <rect>
     <x>561</x>
     <y>118</y>
     <width>121</width>
     <height>116</height>
    </rect>
   </property>
   <property name="title">
    <string>角速度(m/s^2)</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_7">
    <item row="0" column="0">
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>ꞷx</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QDoubleSpinBox" name="spinwx"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string>ꞷy</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDoubleSpinBox" name="spinwy"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_17">
      <property name="text">
       <string>ꞷz</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QDoubleSpinBox" name="spinwz"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>11</y>
     <width>360</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>ID号</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="editID">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>刷新ID</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_9">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>49</y>
     <width>111</width>
     <height>60</height>
    </rect>
   </property>
   <property name="title">
    <string>航路选择</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QComboBox" name="cmbFlightPath"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>240</y>
     <width>195</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QPushButton" name="btnCancel">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnOK">
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
